.github/workflows/nightly-rad-CLI-tests.yaml

```diff
@@ -38,15 +41,20 @@ jobs:
             arch: amd64
             file: rad
             ext: .exe
+    permissions:
+      contents: read
     steps:
       - name: Checkout code
         uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+        with:
+          persist-credentials: false
 
       - name: Test CLI download
         run: make test-cli-download OS=${{ matrix.os }} ARCH=${{ matrix.arch }} FILE=${{ matrix.file }} EXT=${{ matrix.ext }}
 
       - name: Create GitHub issue on failure
         if: ${{ failure() }}
+        run:
```

Comment:
does this need a `run: |` or does it work like this

Reply:
good catch! my typo